README.md

```diff
@@ -1 +1,35 @@
-# ethics-in-data-viz
\ No newline at end of file
+# Ethics in Data Visualization workshop
+
+## Introduction
+This hour-long workshop (undergrad and grad level) provides an introduction to ethics in data visualization. Please feel free to use this workshop, repurpose and remix this content, add suggestions to this project, etc.!
+
+This work is part of the IMLS-funded [Visualizing the Future](visualizingthefuture.github.io/) project.
+
+## How to teach this course
+
+We provide two starting points for teaching Ethics in Data Visualization:"
+
+### 1. A single, self-contained instructor slideshow 
+
+This slideshow includes everything you'll need to lead the workshop. In addition to learner-facing slides, there are also hidden instructor slides, speaker notes, etc.
+
+
+### 2. Directories with extra materials
+
+We've included a variety of materials to make in-person instruction easier, and also to help you excerpt some of the activities and ideas here for use in another course. These include:
+* Printable activity guides
+* Other things!!
+
+
+## The goals for this workshop
+
+By the end of this sessions, learners will be able to:
+
+* List the five steps in the data visualization creation process: select topic, problem, or question; acquire data; clean data; analyze data; create the visualization
+* Understand how decisions in the steps up to data visualization can affect the final visualization 
+* Identify common sources of bias in the data visualization process
+* Practice ethics-centered data visualization design
+
+
```

Comment:
Add section that specifies the suggested times for different parts of the workshop:

Lecture: 15-20 min lecturing, including individual/small group reflection questions
1st Activity - Mississippi Census maps (full group): 15-20 min
2nd Activity - Economist makeovers (small group): 30-50 min